avahi-python/avahi-discover/avahi-discover.py

```diff
@@ -209,10 +209,10 @@ def browse_domain(self, interface, protocol, domain):
             self.new_service_type(interface, protocol, self.stype, domain)
 
     def new_domain(self,interface, protocol, domain, flags):
-        if self.zc_ifaces.has_key((interface,protocol)) == False:
+        if ((interface,protocol) in self.zc_ifaces) == False:
             ifn = self.get_interface_name(interface, protocol)
             self.zc_ifaces[(interface,protocol)] = self.insert_row(self.treemodel, None, ifn,None,interface,protocol,None,domain)
-        if self.zc_domains.has_key((interface,protocol,domain)) == False:
+        if ((interface,protocol,domain) in self.zc_domains) == False:
```

Comment:
use `not in` like above

Reply:
I think they do the same thing but your suggestion is better.